### PR TITLE
handle different format of return values

### PR DESCRIPTION
**Pull request by MDunitz**
## Need
handle different format of return values from get_bundles_all pagination endpoint

## Approach
try except to handle alternative return format

## Testing
- tested etl job in query service

### CODE REVIEW

File: dcplib/etl/__init__.py [hunk 1]
```diff
@@ -98,7 +98,12 @@ def extract(self, query=None, max_workers=512, max_dispatchers=1, page_size=500,
                             f"({extracted_bundle_count/total_bundles:.1%}, {error_bundle_count} errors)")
                 futures = []
                 extracted_results = []
-                for bundle in page.get('bundles', page["results"]):
+                try:
+                    bundles = page["results"]
+                except KeyError:
+                    bundles = page["bundles"]
+
+                for bundle in page.get('bundles', bundles):
```
Comment (calvinnhieu) [inline dcplib/etl/__init__.py]: You could also consider a ternary like:
```
bundles = page['bundles'] if 'bundles' in page else page['results']
for bundle in bundles:
```

**Comment by MDunitz** [comment]
@kislyuk I'm not sure I understand your comment. The goal of the pr is just to handle the different data formats and have the same behavior.
the data `dss_client.post_search.paginate(es_query=query, replica=replica, per_page=page_size)` contains the bundle uuids and versions as dicts in an array as the value for the `results` key            while `dss_client.get_bundles_all.paginate(replica=replica, per_page=page_size)` contains the bundle uuids and versions as dicts in an array as the value for the `bundles` key

File: dcplib/etl/__init__.py (hunk 1 above)
Comment (MDunitz) [reply dcplib/etl/__init__.py]: 👍  thanks

**Comment by kislyuk** [comment]
Sorry, I wasn't thinking about this correctly. Your change is right (and Calvin's suggestion makes it better).

**Comment by MDunitz** [comment]
Tests are failing due to an issue with the setting and then retrieving a secret from aws for python3.6 (unrelated to changes in this pr)
```
ERROR: test_value_of_existing_secret_returns_value (tests.test_aws_secret.TestAwsSecret)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/HumanCellAtlas/dcplib/tests/test_aws_secret.py", line 87, in test_value_of_existing_secret_returns_value
    self.assertEqual(secret.value, existing_secret.value)
  File "/home/travis/build/HumanCellAtlas/dcplib/dcplib/aws_secret.py", line 32, in value
    raise RuntimeError("No such secret: {}".format(self.name))
RuntimeError: No such secret: dcplib/test/test_secret/06f6c743-aca4-42b0-bd4c-10ad863ec57f
======================================================================
ERROR: test_when_item_is_in_config_but_not_in_env_and_use_env_is_set_we_use_config (tests.test_config.TestConfig)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/HumanCellAtlas/dcplib/tests/test_config.py", line 46, in tearDown
    self.aws_secret.delete()
  File "/home/travis/build/HumanCellAtlas/dcplib/dcplib/aws_secret.py", line 64, in delete
    raise RuntimeError("No such secret: {}".format(self.name))
RuntimeError: No such secret: dcp/bogo_component/bogo_env_b4e10b94-c203-47a4-a7e8-f0578a842e2f/secrets
```

**Pull request merged**